From a21164f89e09e84f50f8954ea4b0f8cc55af2d0b Mon Sep 17 00:00:00 2001
From: Francisco Maestre <francisco@maestretorreblanca.com>
Date: Mon, 1 Jun 2026 01:08:03 +0200
Subject: [PATCH 05488/15333] speakup: speakup_soft: fix comment style and
 repeated word

Fix comment style issues in speakup_soft.c:

- Move the closing '*/' of the block comment to its own line, as
  required by the kernel coding style

Signed-off-by: Francisco Maestre <francisco@maestretorreblanca.com>
Signed-off-by: Samuel Thibault <samuel.thibault@ens-lyon.org>
Link: https://patch.msgid.link/20260531230804.254962-15-samuel.thibault@ens-lyon.org
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/accessibility/speakup/speakup_soft.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/accessibility/speakup/speakup_soft.c b/drivers/accessibility/speakup/speakup_soft.c
index 6549bfb96e7f..57d83b82f1d1 100644
--- a/drivers/accessibility/speakup/speakup_soft.c
+++ b/drivers/accessibility/speakup/speakup_soft.c
@@ -447,7 +447,8 @@ static int softsynth_adjust(struct spk_synth *synth, struct st_var_header *var)
 		return 0;
 
 	/* We want to set the speech synthesis punctuation level
-	 * accordingly, so it properly tunes speaking A_PUNC characters */
+	 * accordingly, so it properly tunes speaking A_PUNC characters
+	 */
 	var_data = var->data;
 	if (!var_data)
 		return 0;
-- 
2.55.0

